octobot_commons/profiles/profile_data.py

```diff
@@ -67,6 +67,13 @@ class TentaclesData:
     config: dict = dataclasses.field(default_factory=dict)
 
 
+@dataclasses.dataclass
+class BacktestingData:
```

Comment:
Should we create dedicated files for backtesting data and trading data ? As the first usage of this file was profile data

Reply:
I rename it BacktestingContext